src/main/java/com/powsybl/openloadflow/equations/Equation.java

```diff
@@ -100,10 +101,23 @@ void initTarget(LfNetwork network, double[] targets) {
                 targets[row] = network.getBus(num).getTargetV();
                 break;
 
+            case BUS_REMOTE_V:
+                targets[row] = network.getBus(num).getRemoteControlSourceBuses()
+                        .stream()
+                        .filter(LfBus::hasVoltageControl)
+                        .findFirst()
+                        .orElseThrow(() -> new IllegalStateException("None of the remote control source buses have voltage control on"))
```

Comment:
Is it a real issue that all the generators with remote control are disabled ? I have this case on Litgrid network.

Reply:
This (normal) case has already been handled in network read part so at this stage we should not go into this part of the code.